front/src/common/Pathfinding/Pathfinding.tsx

```diff
@@ -339,8 +340,8 @@ function Pathfinding({ zoomToFeature }: PathfindingProps) {
     <div className="pathfinding-in-progress">
       <div className="pathfinding-in-progress-card">
         <div className="pathfinding-in-progress-title-container">
-          <div className="spinner-border" role="status">
-            <span className="sr-only">Loading...</span>
+          <div>
+            <Spinner />
           </div>
```

Comment:
```suggestion
          <Spinner />
```

Reply:
This div is needed, otherwise the spinner is not displayed correctly

Reply:
Fixed as you asked me ! :)